### ISSUE TITLE
[Issue] nym-client fails to run due to missing libssl.so.1.1 on Ubuntu 22.04

### ISSUE THREAD

**Issue by rachyandco** [nymtech/nym#1780]
**Describe the issue**
nym-client fails to  start.
the following error is printed: `libssl.so.1.1: cannot open shared object file'

**Expected behaviour**
nym-client should just start

**Workaround**
```
wget http://debian.mirror.ac.za/debian/pool/main/o/openssl/libssl1.1_1.1.1o-1_amd64.deb
sudo dpkg -i libssl1.1_1.1.1o-1_amd64.deb
```

**Stack Traces**
 `libssl.so.1.1: cannot open shared object file'

**Steps to Reproduce**

- Start a fresh Ubuntu 22.04
- copy nym-client
- start nym-client

**Which area of Nym were you using?**
 - Application: nym-client
 - OS: ubuntu 22.04
 - Version: v1.1.0

**Comment by sven-hash** [comment]
Any news on this ?

**Comment by tommyv1987** [comment]
Hi guys, 

I think this is being openly looked at right now, and hopefully we should have something more concrete soon.

When we use docker to run the nym-client in our test suites we run the following, and everything is okay.

```
FROM --platform=linux/amd64 ubuntu:22.04

ARG UID=1000
ARG GID=1000
ARG NYM_CLIENT_GIT_TAG=nym-binaries-v1.1.16

RUN apt update \
        && apt -y install ca-certificates jq curl vim wget rsync

# Set up a nym user
RUN groupadd -g "${GID}" nym \
        && useradd --create-home --no-log-init -u "${UID}" -g "${GID}" nym

WORKDIR /home/nym

RUN wget https://github.com/nymtech/nym/releases/download/nym-binaries-v1.1.19/nym-client
RUN chmod u+x nym-client

# Ubuntu 22.04 uses libssl3 instead of 1.1 that nym-client is built with. This is a quick fix for that so that we don't need to recompile it.
RUN wget http://nz2.archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_amd64.deb
RUN ls | grep *.deb
RUN dpkg -i libssl1.1_1.1.1f-1ubuntu2_amd64.deb

RUN mkdir ".nym"
RUN chown -R nym:nym ./

VOLUME /home/nym/.nym

COPY .env .

COPY start.sh .
RUN chmod u+x *.sh

ENTRYPOINT ["./start.sh"]
```

Along with a compose-file 


```
version: "3.7"

services:
  nym-client:
    build:
      context: ./
      args:
        NYM_CLIENT_GIT_TAG: develop <replace accordingly>
      tags:
        - nym_client:develop
    user: nym
    image: nym-client:develop
    container_name: nym-client 
    network_mode: "host"
    volumes:
      - "nym_client_volume:/home/nym/.nym"

volumes:
  nym_client_volume:

```
And the `start.sh`: <replace accordingly - where you see fit>
```
#!/usr/bin/env bash

set -x

NYM_CLIENT_ID="$(echo $RANDOM | md5sum | head -c 20)"


if [ ! -d "/home/nym/.nym/clients" ]
then
    ./nym-client --config-env-file .env init --id "${NYM_CLIENT_ID}" 
    sleep 5
else
    NYM_CLIENT_ID=$(basename /home/nym/.nym/clients/*)
fi


# start the nym-client
./nym-client --config-env-file .env run --id "$NYM_CLIENT_ID"
```

**Comment by rachyandco** [comment]
> # Ubuntu 22.04 uses libssl3 instead of 1.1 that nym-client is built with. This is a quick fix for that so that we don't need to recompile it.
> RUN wget http://nz2.archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_amd64.deb
> RUN ls | grep *.deb
> RUN dpkg -i libssl1.1_1.1.1f-1ubuntu2_amd64.deb

this is obviously integrating the known workaround

**Comment by sven-hash** [comment]
This is what we are using too, it working good in between to be fixed

**Comment by octol** [comment]
This one should be fixed by #3504
